src/System.IO.Ports/tests/SerialPort/GetPortNames.cs

```diff
@@ -46,7 +31,33 @@ private void OpenEveryPortName()
                 }
             }
         }
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/20588 - GetPortNames() has registry dependency.", TargetFrameworkMonikers.UapAot)]
+        private void AllHelperPortsAreInGetPortNames()
+        {
+            string[] serialPortNames = SerialPort.GetPortNames();
```

Comment:
Would it be helpful in both cases (in the failing assert message) to list the ports returned from each one? May help see the problem one day.